src/Adapter/Configuration/LogsConfiguration.php

```diff
@@ -90,13 +75,13 @@ public function updateConfiguration(array $configuration)
                 $nbInvalidEmails = count($invalidEmails);
 
                 if ($nbInvalidEmails > 1) {
-                    $errors[] = $this->translator->trans(
+                    $errors[] = $translator->trans(
```

Comment:
Once you have injected the translator in your constructor (like it used to be), you can use `$this->translator` again 👍 

```suggestion
                    $errors[] = $this->translator->trans(
```

Reply:
(same everywhere you wrote `$translator->trans(` )